### PR TITLE
Refactor flash and error messages

### PR DESCRIPTION
**Pull request by FriedSock**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
-  def list_errors(object, field_name)
-    if object.errors.any?
-      unless object.errors.messages[field_name].blank?
-        content_tag :ul do
-          object.errors.messages[field_name].collect do |field|
-            concat(content_tag(:li, field))
-          end
-        end
-      end
+  def flash_message key, msg
```
Comment (pixeltrix) [inline app/helpers/application_helper.rb]: What's our standard on using parentheses with method definitions?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+    content_tag(:div, class: "#{key}-summary") do
+      content_tag :p, msg
+    end
+  end
+
+  def object_error_messages active_model_messages
+    content_tag(:ul) do
+      active_model_messages.map do |field_name, field_messages|
+        errors_for_field field_name, field_messages
```
Comment (pixeltrix) [inline app/helpers/application_helper.rb]: will concating here not get added to the tag?

**Comment by pixeltrix** [comment]
Other than the minor points :+1:

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (ansonK) [inline app/helpers/application_helper.rb]: My preference is to use parentheses for method definitions

**Comment by chriscarter90** [comment]
Crystal likes this. :+1:

**Pull request merged**